modules/distr-tracing-architecture.adoc

```diff
@@ -25,6 +25,21 @@ This module included in the following assemblies:
 
 ** *Jaeger Console* – With the {JaegerName} user interface, you can visualize your distributed tracing data. On the Search page, you can find traces and explore details of the spans that make up an individual trace.
 
+* *{TempoName}* - This component is based on the open source link:https://github.com/grafana/tempo/[Grafana Tempo project].
+
+** *Gateway* – The Gateway handles authentication, authorization and forwards requests to the Distributor or Query frontend service.
+
+** *Distributor* – The Distributor accepts spans in multiple formats including Jaeger, OpenTelemetry, Zipkin. It routes spans to ingesters by hashing the `+traceID+` and using a distributed consistent hash ring.
+
+** *Ingester* – The Ingester batches trace into blocks, creates bloom filters and indexes, and then flushes it all to the backend. Blocks in the backend are generated in the following layout.
```

Comment:
```suggestion
** *Ingester* – The Ingester batches a trace into blocks, creates bloom filters and indexes, and then flushes it all to the backend. Blocks in the backend are generated in the following layout.
```
Also, as a reader, I could not find any `following layout`.

Reply:
I am removing it it's from the grafana website